arviz/tests/test_plots_bokeh.py

```diff
@@ -0,0 +1,93 @@
+"""Tests use the 'bokeh' backend."""
+# pylint: disable=redefined-outer-name,too-many-lines
+from pandas import DataFrame
+import numpy as np
+import pytest
+
+from .helpers import (  # pylint: disable=unused-import
+    eight_schools_params,
+    models,
+    create_model,
+    multidim_models,
+    create_multidimensional_model,
+)
+from ..rcparams import rcParams, rc_context
+from ..plots import (
+    plot_trace,
+    plot_kde,
+    plot_dist,
+)
+
+rcParams["data.load"] = "eager"
+
+
+@pytest.fixture(scope="module")
+def data(eight_schools_params):
+    data = eight_schools_params
+    return data
+
+
+@pytest.fixture(scope="module")
+def df_trace():
+    return DataFrame({"a": np.random.poisson(2.3, 100)})
+
+
+@pytest.fixture(scope="module")
+def discrete_model():
+    """Simple fixture for random discrete model"""
+    return {"x": np.random.randint(10, size=100), "y": np.random.randint(10, size=100)}
+
+
+@pytest.fixture(scope="module")
+def continuous_model():
+    """Simple fixture for random continuous model"""
+    return {"x": np.random.beta(2, 5, size=100), "y": np.random.beta(2, 5, size=100)}
+
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {},
+        {"var_names": "mu"},
+        {"var_names": ["mu", "tau"]},
+        {"combined": True},
+        {"compact": True},
+        {"combined": True, "compact": True, "legend": True},
+        {"lines": [("mu", {}, [1, 2])]},
+        {"lines": [("mu", {}, 8)]},
+    ],
+)
+def test_plot_trace(models, kwargs):
+    axes = plot_trace(models.model_1, backend="bokeh", show=False, **kwargs)
```

Comment:
Aw I missed chance to comment but in future tests I dont think we should test plotting at the user interface anymore. We should import the bokeh plot function and test against those. That way we're speed up tests and making tests more unit like.

This is small amount of tests so no big deal, Ill draw a picture and/or make a PR later to refactor :)

Thank you for adding!

Reply:
Yeah we can change this, I just wanted tests quickly to master and there was also small bugs, same with the current PR for rug.

Let's clean these after initial implementation.